### PR TITLE
Add BTT Kraken

### PR DESCRIPTION
**Pull request by Benoitone**
Add BTT Kraken Board template to Klippain

### CODE REVIEW

**Comment by Surion79** [comment]
please wait until Frix has released a new version. this PR holds a ton of additional commits not related to the PR

**Comment by Benoitone** [comment]
> please wait until Frix has released a new version. this PR holds a ton of additional commits not related to the PR

It's based on develop branch.

File: user_templates/mcu_defaults/main/BTT_Kraken_v1.0.cfg [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+#--------------------------------------#
+#### BTT Octopus MCU definition ########
+#--------------------------------------#
```
Comment (Frix-x) [inline user_templates/mcu_defaults/main/BTT_Kraken_v1.0.cfg:4]:
```suggestion
#--------------------------------------#
##### BTT Kraken MCU definition ########
#--------------------------------------#
```

**Review by Frix-x — CHANGES_REQUESTED** [review]

**Comment by Frix-x** [comment]
Just a slight change needed, but beside that I'm ready to merge

**Review by Frix-x — APPROVED** [review]

**Pull request merged**